### DISCUSSION TITLE
[灌水]正式入驻绳网辣

### DISCUSSION THREAD

**Discussion by THXJ3247** [share121/inter-knot#33] [其他]
老BEE已经成功注册了绳网账号，期待第一个委托[doge]
![AL](https://github.com/user-attachments/assets/34a36b19-eb96-47c2-8b3b-dacf5aa12e47)

**Comment by share121** [comment]
你的图片没有上传成功，你需要等图片显示出完整的 URL 时，才能提交。

**Comment by THXJ3247** [comment]
没想到大名鼎鼎的老BEE也入驻绳网了，看来继法厄同之后的又一个传奇绳匠就要诞生了！！！！[doge]

**Comment by Xianyu1148** [comment]
+1
